tests/console/setup_libyui_running_system.pm

```diff
@@ -27,7 +27,19 @@ use YuiRestClient;
 
 sub run {
     select_console 'root-console';
-    YuiRestClient::setup_libyui_running_system();
+    zypper_call('in libyui-rest-api');
+
+    my $app  = YuiRestClient::get_app();
+    my $port = $app->get_port();
+    record_info('SERVER', "Used host for libyui: " . $app->get_host());
+    record_info('PORT', "Used port for libyui: " . $app->get_port());
+    # Add the port to permanent config and restart firewalld to apply the changes immediately.
+    # This is needed, because if firewall is restarted for some reason, then the port become
+    # closed (e.g. it was faced while saving settings in yast2 lan) and further tests will not
+    # be able to communicate with YaST modules.
+    assert_script_run("firewall-cmd --zone=public --add-port=$port/tcp --permanent");
+    assert_script_run('firewall-cmd --reload');
+    $app->check_connection(timeout => 30, interval => 2);
```

Comment:
Didn't we figure out that app is not running here yet, so we cannot connect to it?